src/pages/collection/Series.tsx

```diff
@@ -106,13 +106,17 @@ const Series = () => {
               <div className="flex gap-x-2">
                 <Link className="font-semibold text-panel-text-primary" to="/webui/collection">Entire Collection</Link>
                 <Icon path={mdiChevronRight} size={1} />
-                <Link
-                  className="font-semibold text-panel-text-primary"
-                  to={`/webui/collection/group/${series.IDs?.ParentGroup}`}
-                >
-                  {group.Name}
-                </Link>
-                <Icon path={mdiChevronRight} size={1} />
+                {group.Size > 1 && (
+                  <>
+                    <Link
+                      className="font-semibold text-panel-text-primary"
+                      to={`/webui/collection/group/${series.IDs?.ParentGroup}`}
```

Comment:
I think `?.ParentGroup` check needs to be moved into `group.Size > 1` condition otherwise you will be rendering a broken link which doesn't make a lot of sense.

Reply:
`group` will be empty if `ParentGroup` doesn't exist because the query will be skipped

Reply:
I mean if you anticipate that `${series.IDs?.ParentGroup}` might be null then just move it to the line 109 and don't try to render a link at all if you are unable to make a valid route anyways.

Reply:
I don't think the optional chaining there is needed as every series will always have a parent group. Its why I added the ``group.Size`` check in the first place since without it, you could go into the group for single series. 

![image](https://github.com/ShokoAnime/Shoko-WebUI/assets/9443295/466e88d5-34d1-45b4-9618-5927dd84b108)

Reply:
There, I fixed it 👍 
`IDs` will always exist so there was no point in putting the `?` anyway